Makefile.toml

```diff
@@ -12,17 +16,17 @@ args = ["clippy", "--all-targets", "--workspace", "--", "-Dwarnings"]
 
 [tasks.ci-build]
 command = "cargo"
-args = ["build", "--all-targets", "--workspace"]
+args = ["build", "--release",  "--all-targets", "--workspace"]
 
 [tasks.ci-test]
 command = "cargo"
-args = ["test", "--lib", "--bins", "--workspace"]
+args = ["test", "--release", "--lib", "--bins", "--workspace"]
```

Comment:
question: Does this still go through all our pull-primes-from-file code? I thought that the release build optimizes out all the test-only configurations but I must be wrong because the tests still run.

Reply:
Good question!

Reply:
Okay. I dug into this more because I also wasn't sure.

So there is three concepts here: "debug" build, "release" build, and the test annotation. The `test` annotation is set when running things under `cargo test` (it doesn't work for intergration tests!). Meanwhile `release` is set when compiling with optimizations on, and `debug` is set when compiling without optimizations. So you can combine `test` with either debug or release builds.